### ISSUE TITLE
Fix for monitor.cpp to support other pool statistics (at the bottom of the display) on boards other than NERDMINER_T_HMI

### ISSUE THREAD

**Issue by ZooloZombie** [BitMaker-hub/NerdMiner_v2#484]
Just a simple fix for monitor.cpp

Line 49
Change #ifdef NERDMINER_T_HMI
into
#if defined NERDMINER_T_HMI || defined ESP32_2432S028R

Same for Line 372
#ifdef NERDMINER_T_HMI 
into
#if defined NERDMINER_T_HMI || defined ESP32_2432S028R

This allows the other board to display the correct pool status other than using the web-based public-pool.io

Without this "fix", it bypasses the whole getPoolAPIUrl function which is what changes the generic public-pool.io into custom API calls for other pools for the bottom display part.

This may need additional defined statements as well for any boards with the display at the bottom for pool information.

**Comment by cosmicpsyop** [comment]
yeah, doable.  i just did not have a copy of that board to test with.  so ...

a couple other notes while reading through my old PRs  ...
1/ it might be possible to include ESP32_2432S028R || ESP32_2432S028_2USB || ESP32_2432S024 || NERDMINER_T_HMI.  but not sure.
2/ The worker data requires a specific API not offered by all sha256 mining sites for `String getPoolAPIUrl(void)`
3/ There is also the small matter of the graphic with the hard coded title bar.   my original motivation was to enable the Umbrel version of public-pool.io, so no change was required.  I tried a hacky solution in the PR but backed it out due to really really long path names was not good.  This was also  recently was raised again here.
https://github.com/BitMaker-hub/NerdMiner_v2/issues/457
4/ The alternate bottom screen view.
